src/rules/fileHeaderRule.ts

```diff
@@ -79,6 +89,18 @@ export class Rule extends Lint.Rules.AbstractRule {
         return [];
     }
 
+    private doesNewLineEndingViolationExist(text: string, offset: number): boolean {
+        let violationExists = false;
+        const entireComment = ts.forEachLeadingCommentRange(
+            text,
+            offset,
+            (pos, end) => text.substring(pos, end + 2));
+
+        violationExists = entireComment !== undefined && entireComment.match(/^.*((\r)?\n){2,}$/mg) !== null;
+
+        return violationExists;
```

Comment:
Looks like you could nix the `violationExists` var and in-line the above expression w/ the return statement.